src/com/twu/biblioteca/MainMenu.java

```diff
@@ -32,36 +34,71 @@ public void showGoodbyeMessage() {
         System.out.println(goodBye);
     }
 
+    public void showBookCheckOutReferenceMessage() {
+        System.out.println(checkOutBookReference);
+    }
+
     public void showOptions() {
-        for(int i = 0; i < options.size(); i++) {
+        for (int i = 0; i < options.size(); i++) {
             System.out.println(options.get(i));
         }
         System.out.print(">");
     }
 
-    public void showList() {
-        for(int i = 0; i < books.size(); i++) {
-            System.out.println(books.get(i));
-        }
-    }
-
     public void manageOptions() {
         int counter = 0;
-        Scanner scanner = new Scanner(System.in);
+        Scanner optionsScanner = new Scanner(System.in);
 
         while (counter == 0) {
-            String userInput = scanner.next();
-            if(userInput.equals("1")) {
-                showList();
-                counter = 1;
-            }
-            else if(userInput.equals("2")) {
+            String userInput = optionsScanner.next();
+            if (userInput.equals("1")) {
+                showBookList();
+                showOptions();
+            } else if (userInput.equals("2")) {
+                showBookList();
+                manageBookList();
+            } else if (userInput.equals("3")) {
                 showGoodbyeMessage();
                 System.exit(0);
-            }
-            else {
+            } else {
                 showInvalidMessage();
             }
         }
     }
+
+    public void showBookList() {
+        for (int i = 0; i < this.books.size(); i++) {
+            Book book = this.books.get(i);
+            if (book.getCheckedOut() == false) {
+                int count = countCheckedOutBooks();
+                System.out.println(String.format(
+                        "%d. %s | %s | %d",
+                        (i + 1) - count, book.getTitle(), book.getAuthor(), book.getYearPublished()));
+            }
+        }
+    }
+
+    public int countCheckedOutBooks() {
+        int count = 0;
+        for (Book b : this.books){
+            if (b.getCheckedOut()) count++;
+        }
+        return count;
+    }
+
+    public void manageBookList() {
+        int count = countCheckedOutBooks();
+      if ((this.books.size() - count) == 0) {
```

Comment:
if `x - y = 0`, then what is the relationship between `x` and `y`?  I ask because there's a clearer way to express this condition.

Reply:
Ahh I'm so silly. It's so basic.

x = y

I should be asking if this.books.size() == count